examples/threaded_example.nit

```diff
@@ -0,0 +1,21 @@
+# test for threaded annotation
+module threaded_example
+
+import pthreads
+import standard::time
+	
+# threaded fun
+fun foo is threaded do
+	for i in 100.times do
+		print "threaded print"
+	end
+end
+
+foo
+for i in 100.times do
+	print "main print"
+	sys.nanosleep(0,3)
+end
+
+
+
```

Comment:
You should move this file with the pthreads examples in lib/pthread/examples/.

Also you should remove the trailing empty lines, and add the copyright text header.

The "print" in the printed text is a bit redundant, only "threaded" and "main" would be enough.